theia-rust-docker/next.package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "private": true,
+    "theia": {
+        "frontend": {
+            "config": {
+                "applicationName": "Theia Rust Example"
+            }
+        }
+    },
+    "dependencies": {
+        "@theia/core": "next",
+        "@theia/cpp-debug": "*",
+        "@theia/editor": "next",
+        "@theia/editorconfig": "next",
+        "@theia/editor-preview": "next",
+        "@theia/file-search": "next",
+        "@theia/filesystem": "next",
+        "@theia/git": "next",        
```

Comment:
[minor] there is some leftover whitespace here. (editing within VSCode)

<div align='left'>

<img width="253" alt="Screen Shot 2019-08-12 at 10 35 52 AM" src="https://user-images.githubusercontent.com/40359487/62873213-00e83600-bced-11e9-80aa-dba4b041aa59.png">


</div>

Reply:
ok